GoogleSignInSwift.podspec

```diff
@@ -0,0 +1,29 @@
+Pod::Spec.new do |s|
+  s.name = 'GoogleSignInSwift'
+  s.version = '0.1.0'
+  s.swift_version = '4.0'
+  s.summary = 'Adds Swift-focused support for Google Sign In.'
+  s.description = <<-DESC
+The Google Sign-In SDK allows users to sign in with their Google account from third-party apps.
+DESC
+  s.homepage = 'https://developers.google.com/identity/sign-in/ios/'
+  s.license = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors = 'Google LLC'
+  s.source = {
+    :git => 'https://github.com/google/GoogleSignIn-iOS.git',
+    :tag => s.version.to_s
+  }
+  ios_deployment_target = '13.0'
+  macos_deployment_target = '10.15' 
+  s.ios.deployment_target = ios_deployment_target
+  s.osx.deployment_target = macos_deployment_target
+  s.prefix_header_file = false
+  s.source_files = [
+    'GoogleSignInSwift/Sources/*.swift',
+  ]
+  s.frameworks = [
+    'CoreGraphics',
+    'SwiftUI',
+  ]
+  s.dependency 'GoogleSignIn', '~> 6.1'
```

Comment:
Update this to 6.2 after rebasing from main to bring in my 6.2 version bump.

Reply:
Sounds good.